### ISSUE TITLE
GST SVT AV1 test fails with upgrading GST to 1.19.1

### ISSUE THREAD

**Issue by dpatel257** [OpenVisualCloud/Dockerfiles#754]
```
gst-launch-1.0 -v filesrc location=test.yuv ! videoparse format=i420 width=320 height=240 framerate=30 ! svtav1enc ! webmmux ! filesink location=test.mkv
gst-launch-1.0 -v filesrc location=test.mkv ! decodebin ! filesink location=test.xx
```

Above test fails with below error:

```
3: ERROR: from element /GstPipeline:pipeline0/GstDecodeBin:decodebin0/GstAV1Dec:av1dec0: Failed to decode frame
3: Additional debug info:
3: ../ext/aom/gstav1dec.c(447): gst_av1_dec_handle_frame (): /GstPipeline:pipeline0/GstDecodeBin:decodebin0/GstAV1Dec:av1dec0
3: ERROR: pipeline doesn't want to preroll.
3: Setting pipeline to NULL ...
3: ERROR: from element /GstPipeline:pipeline0/GstDecodeBin:decodebin0/GstAV1Dec:av1dec0: Failed to decode frame
3: Additional debug info:
3: ../ext/aom/gstav1dec.c(447): gst_av1_dec_handle_frame (): /GstPipeline:pipeline0/GstDecodeBin:decodebin0/GstAV1Dec:av1dec0
3: ERROR: pipeline doesn't want to preroll.
3: Freeing pipeline ...
1/1 Test #3: test_xeon_gst_ubuntu1804_gst_svt ...***Failed    9.43 sec
```

**Comment by dpatel257** [comment]
Jun: The error from https://github.com/GStreamer/gst-plugins-bad/blob/1.19.1/ext/aom/gstav1dec.c#L441 caused the test case to fail. It is in AV1 decoder which we don’t own. 

/test/gst_svt.sh is to be updated to fix the failing test.

**Comment by dpatel257** [comment]
Fixed with #750 & #751
